docs/HyperIndex/Advanced/hypersync.md

```diff
@@ -26,12 +26,12 @@ Don't see your network here? Pop us a message in [Discord](https://discord.gg/Q9
 | Ethereum Mainnet | 1            |
 | Goerli           | 5            |
 | Optimism         | 10           |
-| Flare            | 14           |
```

Comment:
Why are you removing flare?

Flare is synced

Reply:
Nice, i removed because of a sheet i saw it was decomm'd: https://discord.com/channels/1093188574305857656/1187712782595002458/1262389032562724927